### ISSUE TITLE
Visual representation of forward models

### ISSUE THREAD

**Issue by eivindsm** [equinor/webviz-ert#94]
A relevant idea for webviz-ert: make a visual representation of the forward models; sequence, dependencies, and status during execution. Idea originally submitted by MOD SOUTH

**Comment by dotfloat** [comment]
Forward models aren't persisted in the ERT Storage as yet. Status is especially difficult as ERT Storage is meant to be persistent, immutable long-term store while execution status is something that changes often.

Also, I might be incorrect, but does dependency information even exist for forward models? I was under the impression that they were defined as a sequence of jobs to be executed and nothing more.

**Comment by eivindsm** [comment]
You are absolutely correct, however efforts with the new ensemble evaluator will hopefully change this in the not so distant future.
